site/docs/master/supported-providers.md

```diff
@@ -0,0 +1,77 @@
+# Providers
+
+Velero can store its backups in any S3 or S3 compatible object storage.
```

Comment:
This doesn't make sense as a headline here since object stores other than S3 are supported, and also this page covers volume snapshot operations as well. I'd go with something more similar to what we already have on the existing page, e.g. "Velero supports a variety of storage providers for different backup and snapshot operations. Velero has a plugin system which allows anyone to add compatibility for additional backup and volume storage platforms without modifying the Velero codebase."

Reply:
Thanks for the explanation.